insonmnia/marketplace/server.go

```diff
@@ -199,6 +196,10 @@ func (m *Marketplace) CancelOrder(_ context.Context, req *pb.Order) (*pb.Empty,
 	return &pb.Empty{}, nil
 }
 
+func (m *Marketplace) GetProcessing(ctx context.Context, req *pb.Empty) (*pb.GetProcessingReply, error) {
+	return nil, nil
```

Comment:
`notImplemented` maybe?

Reply:
It exists just for matching the Marketplace interface. The Market service itself unable to know anything about processing orders, this method is for the Node only.

Reply:
But you're right, I must place some note about this into the code.